### PR TITLE
+ improves conductr url setting from the console, and smater default

### LINKED ISSUE #44: Default conductr ip should use CONDUCTR_IP as the CLI does
The [Conduct CLI](https://github.com/typesafehub/typesafe-conductr-cli#conduct) uses `CONDUCTR_IP:CONDUCTR_PORT` as default values instead of assuming localhost.
To be consistent among the projects the sbt plugin should do so as well.

Here's the spot: https://github.com/sbt/sbt-typesafe-conductr/commit/62723f21587cf0bb02c0800bbd3aeeec4d6594c5#commitcomment-10347447

### PR DESCRIPTION
**Pull request by ktoso**
Resolves #44 #46

### CODE REVIEW

File: src/test/scala/com/typesafe/conductr/sbt/SbtTypesafeConductRSpec.scala [hunk 1]
```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright © 2014 Typesafe, Inc. All rights reserved.
+ */
+
+package com.typesafe.conductr.sbt
+
+import akka.actor.ActorSystem
+import org.scalatest.{ BeforeAndAfterAll, Matchers, PrivateMethodTester, WordSpec }
+
+class SbtTypesafeConductRSpec extends WordSpec with Matchers with BeforeAndAfterAll with PrivateMethodTester {
```
Comment (huntc) [inline src/test/scala/com/typesafe/conductr/sbt/SbtTypesafeConductRSpec.scala]: Whoa - I didn't know about `PrivateMethodTester`. Neato.

Comment (ktoso) [inline src/test/scala/com/typesafe/conductr/sbt/SbtTypesafeConductRSpec.scala]: Quite hacky, I'd rather `private[conductr]`, but followed style of tests around here

File: src/main/scala/com/typesafe/conductr/sbt/SbtTypesafeConductR.scala [hunk 2]
```diff
@@ -217,6 +238,13 @@ object SbtTypesafeConductR extends AutoPlugin {
       }
     }
 
+  private def envConductrUrl(): Option[URL] = {
+    val ip = sys.env.get("CONDUCTR_IP")
+    val port = sys.env.getOrElse("CONDUCTR_PORT", 9005)
+
+    ip map { i => new URL(s"http://$i:$port") }
```
Comment (hseeberger) [inline src/main/scala/com/typesafe/conductr/sbt/SbtTypesafeConductR.scala]: No infix, please.

Comment (ktoso) [inline src/main/scala/com/typesafe/conductr/sbt/SbtTypesafeConductR.scala]: updated the pr

**Comment by huntc** [comment]
LGTM

**Pull request merged**